tests/test_buffer_dynamic.py

```diff
@@ -97,6 +161,7 @@ def test_changeCableLen(self, dvs, testlog):
         dvs.runcmd("config interface cable-length Ethernet0 " + self.cableLenTest1)
         expectedProfile = self.make_lossless_profile_name(self.originalSpeed, self.cableLenTest1)
         self.app_db.wait_for_entry("BUFFER_PROFILE_TABLE", expectedProfile)
+        self.check_new_profile_in_asic_db(dvs, expectedProfile)
         self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:3-4", {"profile": "[BUFFER_PROFILE_TABLE:" + expectedProfile + "]"})
 
         # Remove the lossless PGs
```

Comment:
after the cable length is changed to the new one, can't we check asic db to ensure that profile is removed after L174?

Reply:
No, we can’t. The old profile has been removed but a new one will be created almost at the same time. It’s very likely that the sairedis will reuse the OID of the removed one. In that case, we can not find a chance to check whether it is removed in asic db